olive/model/hf_mappings.py

```diff
@@ -23,6 +23,16 @@
     "translation": "seq2seq-lm",
 }
 
+# mapping from feature to peft task type
```

Comment:
>peft task type [](http://example.com/codeflow?start=26&length=14)

could we put a link here for the supported peft task types?

Reply:
Done!